.gitignore

```diff
@@ -14,3 +14,4 @@ Gemfile.lock
 .ruby-gemset
 *.retry
 *.pyc
+user_playbooks
```

Comment:
It's just a way to commit an empty directory to a git repo. You could also ignore `user_playbooks/*.yml` and add a `user_playbooks/playbook1.yml.example`. I just don't want our users to have to create the `user_playbooks` folder.